CMakeLists.txt

```diff
@@ -170,17 +196,14 @@ install(
   DESTINATION lib/${PROJECT_NAME})
 endif()
 
-ament_export_dependencies(rosidl_default_runtime)
-ament_export_include_directories(include)
-ament_export_libraries(
-  rl_lib
-)
-
 # Install params config files.
 install(DIRECTORY
   params
   launch
   DESTINATION share/${PROJECT_NAME}
 )
 
+ament_export_dependencies(rosidl_default_runtime)
```

Comment:
To be safe, you should just export all the dependencies, again referencing to the `nav2_planner` I wrote